pkg/generator/main.go

```diff
@@ -30,7 +30,7 @@ const (
 
 	// These values must be updated during the release process
 	bundleVersion = ""
-	approvalLink  = ""
+	approvalLink  = "https://github.com/kubernetes-sigs/network-policy-api/issues/90"
```

Comment:
nit this step is not documented anywhere, the fact that the api-approved annotation and this approval Link must be set to an issue number for release that needs to be opened...?
and we don't need link any for the bundle version?

Reply:
This is in the release.md 

```
- Update `pkg/generator/main.go` with the new semver tag and any updates to the API review URL.
- Run the following command `BASE_REF=vmajor.minor.patch make generate` which
  will update generated docs and webhook with the correct version info. (Note
  that you can't test with these YAMLs yet as they contain references to
  elements which wont exist until the tag is cut and image is promoted to
```

Reply:
Yeah we don't have "opening the API Review Issue" in here though